### PR TITLE
Adding basic comfyui truss

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:10]: Use H3 instead of a list here because the list styles are making the image alignment weird

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:17]: Consider moving this section down to right before invocation to make things flow better, or at least mention here that the file we're creating will be used during model invocation

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:28]: They'll have to create the file, not just open it

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:43]:
```suggestion
In this case, I have 2 models: SDXL and a controlnet. Each model needs to have 2 things, `url` and `path`. The `url` is the location for downloading the model. The `path` is where this model will get stored inside the Truss. For the path, follow the same guidelines as used in ComfyUI. Models should get stored inside `checkpoints`, controlnets should be stored inside `controlnet`, etc.
```

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:290]: Please use the new production endpoint: https://docs.baseten.co/api-reference/production-predict

File: comfyui-truss/README.md
Comment (philipkiely-baseten) [inline comfyui-truss/README.md:292]: You'll also have a new response format with the new endpoint, so you can drop `.get("model_output")` from this

**Review by philipkiely-baseten — COMMENTED** [review]
@htrivedi99 this is super cool! I took a look at the content and provided a few suggestions, I'll leave it to @bolasim to approve based on the code.

I'd definitely like to collaborate on a blog post about this topic once this PR is merged into the model library.

File: comfyui-truss/README.md
Comment (htrivedi99) [reply comfyui-truss/README.md:13]: Hi @philipkiely-baseten, thanks for your suggestions! I've modified the Readme based on your feedback. I would love to collaborate with you on a blog post for this once Bola gives the green light.

File: comfyui-truss/README.md
Comment (bolasim) [inline comfyui-truss/README.md:72]: I wonder if we should move the workflow to be `data/` in the truss as well instead of input. What do you think?

File: comfyui-truss/config.yaml
Comment (bolasim) [inline comfyui-truss/config.yaml:1]: drop unused keys from the config.

File: comfyui-truss/config.yaml
Comment (bolasim) [inline comfyui-truss/config.yaml:4]: please add a description

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:30]: why is this defined inside the init?

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:46]: can you move these helper functions to a `helpers.py` file so that the model class is more readable?

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:78]: this can also go into helpers.py

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:80]: should we pin a specific commit here for stability?

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:166]: what could this make this false? Should we just do this in the load?

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:188]: you may want to break and return a meaningful error here. consider using `logging.error` or `logging.exception` instead of print

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:184]: could probably refactor this part into a function

**Review by bolasim — COMMENTED** [review]
lgtm overall. Just a few potential improvements on the readability and maintainability side. Almost there.

File: comfyui-truss/model/model.py
Comment (bolasim) [inline comfyui-truss/model/model.py:46]: nit: `self._data_dir / "comfy_ui_workflow.json"`

File: comfyui-truss/model/model.py
Comment (bolasim) [reply comfyui-truss/model/model.py:57]: I still don't understand why not do this in the load?

**Review by bolasim — APPROVED** [review]
lgtm overall. I think the websocket should be opened in the load, not during the predict if possible.

**Pull request merged**
